README.md

```diff
@@ -101,14 +101,14 @@ import (
 	"log"
 	"os"
 
-	"github.com/replicatedhq/replicated/client"
+	"github.com/replicatedhq/replicated/pkg/platform_client"
```

Comment:
I think underscores are discouraged in package names https://blog.golang.org/package-names